spec/support/capybara.rb

```diff
@@ -45,7 +45,7 @@ def self.capybara_tmp_path
 
 # used without docker
 Capybara.register_driver :selenium_chrome_headless do |app|
-  options.add_argument("--headless")
+  options.add_argument("--headless") unless ENV.fetch("HEADED", false)
```

Comment:
kept commenting/uncommenting this line to debug in a browser... much easier for developers to have a CLI flag to use (`HEADED=1 rspec spec/...`)

Reply:
ahh! cool! will remove this!